### PR TITLE
keymap fixes

### PR DESCRIPTION
**Pull request by JeromeSchmied**
- **fix(keymaps): helps are now more consistent, keymaps are better IMO**
- **fix(keymaps): removed x, removed unnecessary Pages::Help**
- **fix(code): applied clippy fixes**
- **fix: better keymaps, keymap help messages**
- **Revert "fix(code): applied clippy fixes"**
- **fix: deleted chess-tui.fen which is not needed**

### CODE REVIEW

File: src/popups.rs [hunk 1]
```diff
@@ -55,7 +55,7 @@ pub fn render_end_popup(frame: &mut Frame, sentence: String) {
         Line::from(sentence).alignment(Alignment::Center),
         Line::from(""),
         Line::from(""),
-        Line::from("Press R to start a new game").alignment(Alignment::Center),
+        Line::from("Press 'R' to start a new game").alignment(Alignment::Center),
```
Comment (thomas-mauran) [inline src/popups.rs:58]:
```suggestion
        Line::from("Press `R` to start a new game").alignment(Alignment::Center),
```

`` maybe to stay consistent with the rest of the modifications

**Review by thomas-mauran — COMMENTED** [review]
LGTM, just need to fix resolve the conflict, thanks for the contribution !

File: src/popups.rs (hunk 1 above)
Comment (JeromeSchmied) [reply src/popups.rs:58]: Well, it's true, but I thought that Esc and Space are special keys, therefore they got the \`\` sorrounding, while R is a simple character, and so I'd say it shall be sorrounded with ''.

But I might as well just change it to be consistent, which one do you prefer?

Comment (thomas-mauran) [reply src/popups.rs:58]: it's really nitpicking but I prefer `` some people might say that R is also a key like `esc` or `space` :D

**Pull request merged**